src/vs/workbench/browser/parts/editor/breadcrumbs.ts

```diff
@@ -123,7 +123,10 @@ Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfigurat
 		'breadcrumbs.enabled': {
 			description: localize('enabled', "Enable/disable navigation breadcrumbs."),
 			type: 'boolean',
-			default: true
+			// --- Start Positron ---
+			// Breadcrumbs are disabled by default in Positron.
+			default: false
+			// --- End Positron ---
```

Comment:
I agree that we should leave this default unchanged until we're ready to make the new action bar on by default.

Reply:
I was certain that on Friday we agreed that we'd turn breadcrumbs off to gauge how people felt about the change. I've reverted this change.